src/easy-sql-test.js

```diff
@@ -50,19 +58,26 @@ export default class EasySQLTest {
 
     request.multiple = true;
     request.verbose = false;
-
     return request.execute(storProcName, callback);
   }
 
   _query(query, callback) {
+    if (!query) {
+      throw 'easy-sql-test: _query() requires query';
+    }
+
     let request = this._connection.request();
     return request.query(query, callback);
   }
 
-  _convertQueriesToTestSteps(prepQueries = []) {
+  _convertQueriesToTestSteps(queries = []) {
     let result = [];
 
-    for (let query of prepQueries) {
+    for (let query of queries) {
+      if (!query) {
```

Comment:
I think we should remove this block. Let `_query()` function to fail and notify developer about it instead of removing a step from the array. Silent removal is way worse then loud fail here